added requirements for exim. Issue #8087 (#8096)
netdata/netdata: collectors/python.d.plugin/exim/README.md

Simple module executing `exim -bpc` to grab exim queue.
This command can take a lot of time to finish its execution thus it is not recommended to run it every second.

## Requirements

> The module uses the `exim` binary, which can only be executed as root by default. We need to allow other users to `exim` binary. We solve that adding `queue_list_requires_admin` statement in exim configuration and set to `false`, because is `true` by default. On many Linux distributions, the default location of `exim` configuration is in `/etc/exim.conf`.

Reviewer: "because **it** is `true` by default."

Author: @shortpatti this is changed and commited. Thank you.

Simple module executing `exim -bpc` to grab exim queue.
This command can take a lot of time to finish its execution thus it is not recommended to run it every second.

# Exim monitoring with Netdata

Simple module executing `exim -bpc` to grab exim queue.

> This command can take a lot of time to finish its execution thus it is not recommended to run it every second.

Reviewer 2: This is a repetition of lines 1-4 just above. You can remove these.

## Requirements

> The module uses `exim` binary which can only be executed as root by default. We need to allow other user that can run `exim` binary. We solving that adding `queue_list_requires_admin` statement in exim configuration and set to `false` because is `true` by default. On many Linux OS default location of `exim` configuration was in `/etc/exim.conf`.

Reviewer 2: ```suggestion
The module uses the `exim` binary, which can only be executed as root by default. We need to allow other users to `exim` binary. We solve that adding `queue_list_requires_admin` statement in exim configuration and set to `false`, because is `true` by default. On many Linux distributions, the default location of `exim` configuration is in `/etc/exim.conf`.
```

> 1. Edit config with favor editor and add

Reviewer 2: ```suggestion
1. Edit the `exim` configuration with your preferred editor and add:
```

`queue_list_requires_admin = false`

> 2. Restart exim and netdata

Reviewer 2: ```suggestion
2. Restart `exim` and Netdata
```

*WHM (CPanel) server*

> On the WHM server `exim` we can reconfigure over WHM interface with next steps.

Reviewer 2: ```suggestion
On a WHM server, you can reconfigure `exmin` over the WHM interface with the following steps.
```

1. Login to WHM
2. Navigate to 

> Service Configuration --> Exim Configuration Manager --> tab Advanced Editor

Reviewer 2: These can be on one line.

> 3. Scroll down to button `Add additional configuration setting`and click on it.

Reviewer 2: ```suggestion
3. Scroll down to the button **Add additional configuration setting** and click on it.
```

4. In the new dropdown which will appear above we need to find and choose:
`queue_list_requires_admin` and set to `false` 

> 5. Scroll to the end and click `Save` button.

Reviewer 2: ```suggestion
5. Scroll to the end and click the **Save** button.
```
